contracts/DaiHrd.sol

```diff
@@ -344,6 +345,7 @@ contract ERC777 is RuntimeConstants, Context, IERC777, IERC20 {
 		emit Transfer(address(0), account, amount);
 	}
 
+	// Note: Keydonix changed visibility from private to internal, we reference this function in derived contract
```

Comment:
I feel like we should have a standardize comment style for these changes so they are easy to find (and thus re-apply) if we need to upgrade ERC777.  `// KEYDONIX: ...` maybe?

Reply:
We should probably also add such a comment to the change we made to protect from the Uniswap bug.